[2023-05-17T16:46:19.134+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [queued]>
[2023-05-17T16:46:19.155+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [queued]>
[2023-05-17T16:46:19.156+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-17T16:46:19.183+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-10 00:00:00+00:00
[2023-05-17T16:46:19.197+0000] {standard_task_runner.py:57} INFO - Started process 1206 to run task
[2023-05-17T16:46:19.202+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-10T00:00:00+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmpuq6mi2ra']
[2023-05-17T16:46:19.207+0000] {standard_task_runner.py:85} INFO - Job 26: Subtask clean_fondue_data
[2023-05-17T16:46:19.295+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [running]> on host 8cf8fb9e28f7
[2023-05-17T16:46:19.473+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-10T00:00:00+00:00'
[2023-05-17T16:46:19.661+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-17T16:46:19.714+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-17T16:46:19.732+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230510T000000, start_date=20230517T164619, end_date=20230517T164619
[2023-05-17T16:46:19.784+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-17T16:46:19.861+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
<<<<<<< HEAD
[2023-05-17T17:55:01.113+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [queued]>
[2023-05-17T17:55:01.132+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [queued]>
[2023-05-17T17:55:01.133+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-17T17:55:01.159+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-10 00:00:00+00:00
[2023-05-17T17:55:01.170+0000] {standard_task_runner.py:57} INFO - Started process 703 to run task
[2023-05-17T17:55:01.173+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-10T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmpix4m9x2x']
[2023-05-17T17:55:01.176+0000] {standard_task_runner.py:85} INFO - Job 25: Subtask clean_fondue_data
[2023-05-17T17:55:01.255+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [running]> on host a37c7bc3731c
[2023-05-17T17:55:01.389+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-10T00:00:00+00:00'
[2023-05-17T17:55:01.497+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-17T17:55:01.556+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-17T17:55:01.571+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230510T000000, start_date=20230517T175501, end_date=20230517T175501
[2023-05-17T17:55:01.627+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-17T17:55:01.723+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-17T18:18:11.273+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [queued]>
[2023-05-17T18:18:11.298+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [queued]>
[2023-05-17T18:18:11.299+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-17T18:18:11.555+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-10 00:00:00+00:00
[2023-05-17T18:18:11.570+0000] {standard_task_runner.py:57} INFO - Started process 846 to run task
[2023-05-17T18:18:11.597+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-10T00:00:00+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmp431hhehv']
[2023-05-17T18:18:11.606+0000] {standard_task_runner.py:85} INFO - Job 26: Subtask clean_fondue_data
[2023-05-17T18:18:12.166+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [running]> on host 3390ff13fbf6
[2023-05-17T18:18:12.667+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-10T00:00:00+00:00'
[2023-05-17T18:18:12.758+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-17T18:18:13.166+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-17T18:18:13.180+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230510T000000, start_date=20230517T181811, end_date=20230517T181813
[2023-05-17T18:18:13.880+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-17T18:18:13.914+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-05-17T18:25:45.059+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [queued]>
[2023-05-17T18:25:45.080+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [queued]>
[2023-05-17T18:25:45.081+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-17T18:25:45.213+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-10 00:00:00+00:00
[2023-05-17T18:25:45.224+0000] {standard_task_runner.py:57} INFO - Started process 772 to run task
[2023-05-17T18:25:45.227+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-10T00:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmpgzn2_9c0']
[2023-05-17T18:25:45.229+0000] {standard_task_runner.py:85} INFO - Job 19: Subtask clean_fondue_data
[2023-05-17T18:25:45.306+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [running]> on host 70fcd41af239
[2023-05-17T18:25:45.424+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-10T00:00:00+00:00'
[2023-05-17T18:25:45.536+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-17T18:25:45.631+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-17T18:25:45.647+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230510T000000, start_date=20230517T182545, end_date=20230517T182545
[2023-05-17T18:25:45.682+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-17T18:25:45.733+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-05-17T18:37:02.715+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [queued]>
[2023-05-17T18:37:02.733+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [queued]>
[2023-05-17T18:37:02.734+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-17T18:37:02.757+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-10 00:00:00+00:00
[2023-05-17T18:37:02.766+0000] {standard_task_runner.py:57} INFO - Started process 655 to run task
[2023-05-17T18:37:02.768+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-10T00:00:00+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmpm6mh8_8g']
[2023-05-17T18:37:02.770+0000] {standard_task_runner.py:85} INFO - Job 18: Subtask clean_fondue_data
[2023-05-17T18:37:02.836+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [running]> on host 1545bed90dab
[2023-05-17T18:37:02.967+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-10T00:00:00+00:00'
[2023-05-17T18:37:03.067+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-17T18:37:03.126+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-17T18:37:03.145+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230510T000000, start_date=20230517T183702, end_date=20230517T183703
[2023-05-17T18:37:03.224+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-17T18:37:03.272+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-05-18T06:41:12.742+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [queued]>
[2023-05-18T06:41:12.765+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [queued]>
[2023-05-18T06:41:12.767+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-18T06:41:12.799+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-10 00:00:00+00:00
[2023-05-18T06:41:12.811+0000] {standard_task_runner.py:57} INFO - Started process 1206 to run task
[2023-05-18T06:41:12.815+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-10T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmpy1cwwom2']
[2023-05-18T06:41:12.818+0000] {standard_task_runner.py:85} INFO - Job 27: Subtask clean_fondue_data
[2023-05-18T06:41:12.905+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [running]> on host 714a0c7ad307
[2023-05-18T06:41:13.205+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-10T00:00:00+00:00'
[2023-05-18T06:41:13.339+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-18T06:41:13.410+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-18T06:41:13.429+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230510T000000, start_date=20230518T064112, end_date=20230518T064113
[2023-05-18T06:41:13.472+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-18T06:41:13.720+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-18T06:53:29.577+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [queued]>
[2023-05-18T06:53:29.597+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [queued]>
[2023-05-18T06:53:29.598+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-18T06:53:29.636+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-10 00:00:00+00:00
[2023-05-18T06:53:29.647+0000] {standard_task_runner.py:57} INFO - Started process 1482 to run task
[2023-05-18T06:53:29.651+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-10T00:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmp40dk1sf9']
[2023-05-18T06:53:29.654+0000] {standard_task_runner.py:85} INFO - Job 35: Subtask clean_fondue_data
[2023-05-18T06:53:29.779+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [running]> on host 26547302378e
[2023-05-18T06:53:30.007+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-10T00:00:00+00:00'
[2023-05-18T06:53:30.247+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-18T06:53:30.368+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-18T06:53:30.401+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230510T000000, start_date=20230518T065329, end_date=20230518T065330
[2023-05-18T06:53:30.473+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-18T06:53:30.531+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
=======
[2023-05-18T04:22:16.006+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [queued]>
[2023-05-18T04:22:16.047+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [queued]>
[2023-05-18T04:22:16.049+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-18T04:22:16.114+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-10 00:00:00+00:00
[2023-05-18T04:22:16.129+0000] {standard_task_runner.py:57} INFO - Started process 476 to run task
[2023-05-18T04:22:16.137+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-10T00:00:00+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmphb2h4a7_']
[2023-05-18T04:22:16.142+0000] {standard_task_runner.py:85} INFO - Job 29: Subtask clean_fondue_data
[2023-05-18T04:22:16.317+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [running]> on host b36de7cfc389
[2023-05-18T04:22:16.612+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-10T00:00:00+00:00'
[2023-05-18T04:22:16.721+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-18T04:22:16.791+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-18T04:22:16.804+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230510T000000, start_date=20230518T042216, end_date=20230518T042216
[2023-05-18T04:22:16.953+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-18T04:22:17.046+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
>>>>>>> 8e4a41b368593e9ce283ae02afd6c27252f67370
[2023-05-18T07:21:06.386+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-10T00:00:00+00:00/task_id=clean_fondue_data permission to 509
[2023-05-18T07:21:06.388+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-10T00:00:00+00:00/task_id=clean_fondue_data permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-10T00:00:00+00:00/task_id=clean_fondue_data'
[2023-05-18T07:21:06.511+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-10T00:00:00+00:00/task_id=clean_fondue_data permission to 509
[2023-05-18T07:21:06.513+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-10T00:00:00+00:00/task_id=clean_fondue_data permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-10T00:00:00+00:00/task_id=clean_fondue_data'
[2023-05-18T07:21:06.585+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [queued]>
[2023-05-18T07:21:06.610+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [queued]>
[2023-05-18T07:21:06.611+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-18T07:21:06.834+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-10 00:00:00+00:00
[2023-05-18T07:21:06.848+0000] {standard_task_runner.py:57} INFO - Started process 1841 to run task
[2023-05-18T07:21:06.854+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-10T00:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmpepni8i2v']
[2023-05-18T07:21:06.859+0000] {standard_task_runner.py:85} INFO - Job 35: Subtask clean_fondue_data
[2023-05-18T07:21:06.981+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-10T00:00:00+00:00/task_id=clean_fondue_data permission to 509
[2023-05-18T07:21:06.984+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-10T00:00:00+00:00/task_id=clean_fondue_data permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-10T00:00:00+00:00/task_id=clean_fondue_data'
[2023-05-18T07:21:06.990+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [running]> on host 0a50b96f7e57
[2023-05-18T07:21:07.285+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-10T00:00:00+00:00'
[2023-05-18T07:21:07.667+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-18T07:21:07.994+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-18T07:21:08.021+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230510T000000, start_date=20230518T072106, end_date=20230518T072108
[2023-05-18T07:21:08.080+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-18T07:21:08.168+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-05-18T07:38:27.893+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-10T00:00:00+00:00/task_id=clean_fondue_data permission to 509
[2023-05-18T07:38:27.895+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-10T00:00:00+00:00/task_id=clean_fondue_data permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-10T00:00:00+00:00/task_id=clean_fondue_data'
[2023-05-18T07:38:27.941+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-10T00:00:00+00:00/task_id=clean_fondue_data permission to 509
[2023-05-18T07:38:27.943+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-10T00:00:00+00:00/task_id=clean_fondue_data permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-10T00:00:00+00:00/task_id=clean_fondue_data'
[2023-05-18T07:38:27.991+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [queued]>
[2023-05-18T07:38:28.018+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [queued]>
[2023-05-18T07:38:28.020+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-18T07:38:28.117+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-10 00:00:00+00:00
[2023-05-18T07:38:28.126+0000] {standard_task_runner.py:57} INFO - Started process 891 to run task
[2023-05-18T07:38:28.130+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-10T00:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmpsyrqq_q6']
[2023-05-18T07:38:28.133+0000] {standard_task_runner.py:85} INFO - Job 28: Subtask clean_fondue_data
[2023-05-18T07:38:28.211+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-10T00:00:00+00:00/task_id=clean_fondue_data permission to 509
[2023-05-18T07:38:28.213+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-10T00:00:00+00:00/task_id=clean_fondue_data permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-10T00:00:00+00:00/task_id=clean_fondue_data'
[2023-05-18T07:38:28.218+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-10T00:00:00+00:00 [running]> on host 4fe746fadf66
[2023-05-18T07:38:28.379+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-10T00:00:00+00:00'
[2023-05-18T07:38:28.529+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-18T07:38:28.657+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-18T07:38:28.675+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230510T000000, start_date=20230518T073827, end_date=20230518T073828
[2023-05-18T07:38:28.745+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-18T07:38:28.811+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
